File: LabCodes/lab2.c
#include <wiringPi.h>
#include <time.h>

int main()
{
	wiringPiSetup();
	pinMode(23, OUTPUT);
    pinMode(24, OUTPUT);
	while(1)
	{
		sleep(1);
		digitalWrite(23, 1);
        digitalWrite(24, 0);
		sleep(1);
        digitalWrite(24, 1);
		digitalWrite(23, 0);
	}
	return 0;
}

File: LabCodes/lab2.py
print('hi')

import RPi.GPIO as GPIO
import time

GPIO.setmode(GPIO.BCM)
GPIO.setup(13, GPIO.OUT)
GPIO.setup(19, GPIO.OUT)

while True:
    GPIO.output(19, GPIO.LOW)
    GPIO.output(13, GPIO.HIGH)
    time.sleep(0.25)
    GPIO.output(13, GPIO.LOW)
    GPIO.output(19, GPIO.HIGH)
    time.sleep(0.25)